include/ec_cmd_log.hrl

```diff
@@ -22,3 +22,10 @@
 -define(EC_WARN, 1).
 -define(EC_INFO, 2).
 -define(EC_DEBUG, 3).
+
+-define(RED,     $r).
+-define(GREEN,   $g).
+-define(YELLOW,  $y).
+-define(BLUE,    $b).
+-define(MAGENTA, $m).
+-define(CYAN,    $c).
```

Comment:
This now makes the definitions public. If they are expected to be internal, they should be in `src/ec_cmd_log.hrl` (or some other file) that the test should be able to include but not third party dependencies,

Reply:
That's true; thanks for the feedback.